curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649f12c7c84ffb02b0fc7563.md

```diff
@@ -0,0 +1,297 @@
+---
+id: 649f12c7c84ffb02b0fc7563
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+Write an `else` statement inside which `rolls` will be incremented by `1`. After that, call the  `rollDice` function.
```

Comment:
```suggestion
Write an `else` statement. In the statement, use the increment operator to increment `rolls` by one. Then call the `rollDice` function.
```

If we are not allowing syntax like `rolls+=1` or `rolls=rolls+1`, we should specify using the increment operator.